ext/curl/interface.c

```diff
@@ -1370,6 +1370,10 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLOPT_TCP_FASTOPEN);
 #endif
 
+#if LIBCURL_VERSION_NUM >= 0x073700 /* Available since 7.55.0 */
+	REGISTER_CURL_CONSTANT(CURLOPT_REQUEST_TARGET);
+#endif
```

Comment:
This will also need handling in _php_curl_setopt.

Reply:
Sorry, fixed!